app/routes/datamart/analysis/forest_change/tree_cover_change.py

```diff
@@ -0,0 +1,182 @@
+from enum import Enum
+from typing import Optional
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+from fastapi.logger import logger
+from fastapi.openapi.models import APIKey
+from fastapi.responses import ORJSONResponse
+from httpx import AsyncClient
+from pydantic import Field, root_validator, ValidationError
+
+from app.authentication.api_keys import get_api_key
+from app.models.orm.api_keys import ApiKey
+from app.models.pydantic.base import StrictBaseModel
+
+
+tree_cover_change_router = APIRouter(
+    prefix="/tree_cover_change"
+)
+
+
+class Gadm(str, Enum):
+    ISO = "iso"
+    ADM0 = "adm0"
```

Comment:
This seems confusing - adm0 and ISO are the same, right?  Should you eliminate one of them (probably ADM0), or add a comment on why you have both of them?

Reply:
I agree. I was never happy with that, either. I think I remember it having something to do with the existing query params (interface) that made it clunky. I'll see what I can do. Thanks for the review!